DQM/PhysicsHWW/src/HypDilepMaker.cc

```diff
@@ -48,33 +48,27 @@ void HypDilepMaker::SetVars(HWW& hww, const edm::Event& iEvent, const edm::Event
   double hypJetMinPtCut = 30.0;
 
   // muon charge
-  vector<int> *mus_charge = new vector<int>;
-  *mus_charge = hww.mus_charge(); 
+  vector<int> mus_charge = hww.mus_charge(); 
```

Comment:
wonder if all these assignments should not be just
const auto & bha = hww.bha();
(was this a CINT macro turned into CMSSW producer or what?)